build.cake

```diff
@@ -741,8 +740,7 @@ void BuildXcodeProject(string projectPath)
 }
 
 // Download json package.
-Task("Externals-Uwp-IL2CPP-Dependencies-BuildNewtonsoftJson")
-    .Does(() =>
+Task("BuildNewtonsoftJson").IsDependentOn("Externals-Uwp-IL2CPP-Dependencies").Does(() =>
```

Comment:
vice versa. `Externals-Uwp-IL2CPP-Dependencies` depends on `BuildNewtonsoftJson`. Currently it doesn't run